Cargo.toml

```diff
@@ -21,7 +21,6 @@ test-bpf = []
 assert_matches = "1.4.0"
 solana-sdk = "=1.6.1"
 solana-program-test = "=1.6.1"
-solana-validator = "=1.6.1"
```

Comment:
You don't need to depend on this package, it also takes a really long time to compile, so best to leave it out